app/models/custom_filter.rb

```diff
@@ -33,7 +33,7 @@ class CustomFilter < ApplicationRecord
   include Expireable
   include Redisable
 
-  enum :action, { warn: 0, hide: 1 }, suffix: :action
+  enum :action, { warn: 0, hide: 1, blur_media: 2 }, suffix: :action
```

Comment:
Since this is going to be exposed in the API... Maybe we could just use `blur` to avoid having an underscore?

Reply:
Fine by me, although I don't see the problem with an underscore.